scripts/plusplus.js

```diff
@@ -0,0 +1,74 @@
+/* 
+ * Custom ++/-- script to handle people ++'ing stuff.
+ * 
+ * Features
+ * 
+ *   Per user rate limiting
+ *   Tracking totals
+ *   
+ */
+
+module.exports = function (robot) {
+    
+    //This initilizes the plusplus object.  So shit is gonna get overridden 
+    //everytime we connect to redis.  I think?
+    robot.brain.on('loaded', function () {
+        robot.brain.plusplus = {
+            users: [],
+            words: []
+        };
+    });
+
+    /**
+     * ++ - Increment points of target
+     * */
+    robot.respond('/(.*)\\+\\+/', function (msg) {
+        var target = msg.match[1];
+        var user = msg.message.user.name;
+        var t;
+        if (clearedTimeout(user)) {
+            t = robot.brain.plusplus.words.target || 0;
+            robot.brain.plusplus.words.target = t + 1;
+            setTimeout(user);
+        }
+    });
+
+    /**
+     * -- - Decrement points of target
+     * */
+    robot.respond('/(.*)\\\-\\\-/', function (msg) {
```

Comment:
These should be `robot.hear` instead of `robot.respond`. The reason is that respond requires `@owlbot` at the head of the message. We want to be able to `bill++` without `@owlbot bill++`

The concolusion of each ++/-- should also end with owlbot returning the current score for a specific item.